x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -225,14 +226,15 @@ export class HeadlessChromiumDriverFactory {
         exit$,
       });
 
+      const factoryLogger = this.callerLogger.clone(['chromium-driver-factory']);
```

Comment:
This is a new logger instance versus the original logger instance that gets used for the driver instance. This cleans up a redundant logging tag that previously was getting spread to the instance's logs.

BEFORE
![image](https://user-images.githubusercontent.com/908371/61985689-32f45b00-afbf-11e9-9eee-9217cc70f9f5.png)


AFTER
![image](https://user-images.githubusercontent.com/908371/61985718-499ab200-afbf-11e9-8b80-6e1deffce1a1.png)

Reply:
We can probably also get rid of the redundant `browser-driver` logging tag. Looks like that was to encompass the actual browser type: i.e, PhantomJS or Chromium. There's no PhantomJS anymore though.